src/TaskRunner.php

```diff
@@ -188,7 +188,7 @@ private function createContainer(InputInterface $input, OutputInterface $output,
      */
     private function createApplication()
     {
-        $application = new Application(self::APPLICATION_NAME, null);
+        $application = new Application(self::APPLICATION_NAME, 'UNKNOWN');
```

Comment:
I'm not entirely sure what this `UNKNOWN` is about but the commit message says that the construction of the application is now fixed, so I am assuming that's true :)

Reply:
@imanoleguskiza 
I am also curious about this ´UNKNOWN´